### DISCUSSION TITLE
Allow blocking of repos and/or users in the "for you" feed.

### DISCUSSION THREAD

**Discussion by rhwood** [community/community#14101] [Other Feature Feedback, Questions, & Ideas]
There are certain repos that I really want nothing more to do with (even if I did contribute to them regularly over the past 10 years) and certain users that I once collaborated with who I really don't want to see their activity anymore (even if I did collaborate with them regularly over the past 10 years). Please let me block those repos and users from the "for you" feed.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by rhwood** [reply]
Unfortunately this filtering merely silences everything; where as I want to specifically be able block certain repos or organizations from that feed (I have discovered useful projects using the feed, so I don't want to disable it).

**Reply by jfullstackdev** [reply]
that's really hard because it's showing pseudo-random recommendations, they would not be able to guess users that don't want to see certain repos

**Reply by rhwood** [reply]
I think it would be really easy for GitHub to allow me to explicitly block specific organizations and repos. I didn't ask for blocking "related" things, just specific thing. As I originally reported -- I left an organization, no longer have anything to do with it, no longer want to be reminded of it, and its the repos I contributed to in that organization that is dominating the feed. Within in the constructs of this feed concept to be, I should be able to explicitly block certain known entities from showing in it.

**Reply by jfullstackdev** [reply]
would blocking orgs and repos qualify ? that is maybe the valid request

how about trying Ignore ? maybe it will work 

![image](https://user-images.githubusercontent.com/47092464/182023557-087bb208-f6a9-4e8d-b09f-77e904b8b492.png)
